projects/transformers/run.py

```diff
@@ -408,15 +461,26 @@ def run_finetuning_multiple_tasks(
         if task_name in model_args.task_hyperparams:
             for hp_key, hp_val in model_args.task_hyperparams[task_name].items():
                 setattr(training_args, hp_key, hp_val)
-        # create Task Results
-        task_results = TaskResults(task_name, training_args=training_args)
+
+        # create TaskResults instance
+        has_early_stopping = False
+        for callback in model_args.trainer_callbacks:
+            if isinstance(callback, EarlyStoppingCallback):
+                has_early_stopping = True
+                break
+
+        task_results = TaskResults(task_name,
+                                   has_early_stopping,
+                                   training_args=training_args)
 
         # Run finetuning and save results
         for _ in range(training_args.num_runs):
             # reset seed per run
-            training_args.seed = random.randint(0, 1000000000)
+            training_args.seed = random.randint(0, 1_000_000_000)
```

Comment:
Huh, interesting, I had no idea 1_000_000_000 was a thing in Python. A lot easier to read.